Header header
string file_name
int32 job_cycles
bool start_job
bool loop # constantly loop the task
int32 start_atline # starts at 0.
